### ISSUE TITLE
How to get comments from specific or single group post?

### ISSUE THREAD

**Issue by ru-jee** [kevinzg/facebook-scraper#368]
Hi.
I've tried to scrape posts and comments from group, but the code stops after 50-60 posts (normally with exit code 0).
Then I've got list of post_ids using selenium and BeautifulSoup.
However, when I trying to use get_posts() method with group post id (post_urls=my_id_list) I've got only post data, not comments (comments_full list is empty). Adding `options={"comments":True}` didn't help.
Is there a way to get single post from FB group with its comments?

**Comment by neon-ninja** [comment]
Try pass cookies or credentials

**Comment by ru-jee** [comment]
> 
> 
> Try pass cookies or credentials

Thanks, but I already used authorisation with cookies or credentials. It didn't help.

**Comment by neon-ninja** [comment]
I see - try this - https://github.com/kevinzg/facebook-scraper/commit/4ebd21d2ddfc6567ecb2388c742545e21b6d3c6e

**Comment by ru-jee** [comment]
> 
> 
> I see - try this - [4ebd21d](https://github.com/kevinzg/facebook-scraper/commit/4ebd21d2ddfc6567ecb2388c742545e21b6d3c6e)

Yep, it works. Thank you for your help and useful tool.

**Comment by Dennis1989** [comment]
Somehow I only get 10 comments (and no replies) per page. Has this something to to with 4ebd21d or is my account simply blocked?

**Comment by neon-ninja** [comment]
Impossible to tell without you posting the code you used to cause this problem

**Comment by Dennis1989** [comment]
Here the code that previously worked (at least I got more comments and some replies). I only want to access one specific post on a specific page and retrieve as many comments and replies as possible. You see, I pickle the result afterward. 

`for post in get_posts(post_urls=[page+"/posts/"+post_id], cookies="./cookies.json", options={"comments": True}):
           pickle.dump(post, file=open("Objects/"+ page +"_"+ post_id +".pickle", "wb"))`

**Comment by neon-ninja** [comment]
Impossible to tell without knowing the value of `page` or `post_id`

**Comment by Dennis1989** [comment]
Basically it is for all posts. As an example this one:
```
page = "cnn"
post = "10162093840556509"
```
The method is always returning approx. 10 (sometimes 12) comments with no replies.

**Comment by neon-ninja** [comment]
I see - try this https://github.com/kevinzg/facebook-scraper/commit/8bcf9893271a7b23fe817cd1009cab29e77dd7ce

**Comment by Dennis1989** [comment]
Thank you for your help. Now replies are crawled but still only just a fraction of comments:

```
DEBUG:facebook_scraper.extractors:Unable to parse am Mo
DEBUG:facebook_scraper.extractors:Unable to parse am Mo
DEBUG:facebook_scraper.extractors:Unable to parse am Mo
DEBUG:facebook_scraper.extractors:Unable to parse am Mo
100%|█████████████████████████████████████████████████████████████████████████| 30/30 [00:12<00:00,  4.48it/s]

481 30
118
```

Is it possible that these parsing errors are due to the encoding?

**Comment by neon-ninja** [comment]
Yes - this looks like a locale issue. Try set your Facebook language to English

**Comment by neon-ninja** [comment]
https://github.com/kevinzg/facebook-scraper/commit/797ba595e53020fed139cd35cb5b6bfcbfb85f89 should make comment pagination work in languages other than English

**Comment by Dennis1989** [comment]
Thank you ninja. Works fine for me now!
